invenio_rdm_records/templates/semantic-ui/invenio_notifications/guest-access-request.submit.jinja

```diff
@@ -9,10 +9,12 @@
     ui=config.SITE_UI_URL, id=request_id
     )
 %}
-{% set account_settings_link = "{ui}/account/settings/notifications".format(
+{# {% set account_settings_link = "{ui}/account/settings/notifications".format(
```

Comment:
i guess this can be removed?

Reply:
yes! thanks for spotting that. Fixed and rebased (fixed conflicts) in passing.

Reply:
en passant :chess_pawn: